### PR TITLE
core(config): include additional trace categories

### PR DESCRIPTION
**Pull request by victorporof**
Signed-off-by: Victor Porof <victorporof@chromium.org>

### CODE REVIEW

File: lighthouse-core/gather/gatherers/trace.js [hunk 1]
```diff
@@ -51,9 +51,12 @@ class Trace extends FRGatherer {
...
 
-      // CPU sampling profiler data only enabled for debugging purposes
-      // 'disabled-by-default-v8.cpu_profiler',
-      // 'disabled-by-default-v8.cpu_profiler.hires',
+      // Additional categories used by devtools. Not used by Lighthouse, but included to facilitate
+      // loading traces from Lighthouse into the Performance panel.
+      'disabled-by-default-devtools.timeline.frame',
+      'disabled-by-default-v8.cpu_profiler',
+      'disabled-by-default-v8.cpu_profiler.hires',
+      'latencyInfo',
```
Comment (brendankenny) [inline lighthouse-core/gather/gatherers/trace.js:59]: how much larger do these typically make traces? Do we need to update things like [`minify-trace`](https://github.com/GoogleChrome/lighthouse/blob/0de46e7c9843cb17c905be2431aec7dd4ffb7e73/lighthouse-core/lib/minify-trace.js) to strip new things (or include new things)?

cc @patrickhulce

Comment (patrickhulce) [reply lighthouse-core/gather/gatherers/trace.js:59]: I would expect `cpu_profiler.hires` to be the largest of these by far on busy pages (which we were planning on enabling anyhow #8526). I think `latencyInfo` can be fairly chatty too if it's what I think it is.

Updating minify-trace to keep what we need from here, would be great! Could use that process to get some rough data on trace sizes for our common culprits (cnn.com, sfgate.com, theverge.com)

**Review by brendankenny — APPROVED** [review]
LGTM, assuming no one else has an issue here.

A run on the verge gives a trace of
current Lighthouse - 50.1MiB
this branch - 60.7MiB

So, it's a significant jump (albeit from an already very large base :). Is that going to be an issue for anyone in terms of raw storage?

We'll definitely want to follow up with `minify-trace` pretty soon, I think.

**Comment by victorporof** [comment]
Filed https://github.com/GoogleChrome/lighthouse/issues/12748 for updating minify-trace.

**Comment by brendankenny** [comment]
> Is that going to be an issue for anyone in terms of raw storage?

Enough time for objections has probably passed :) we can just keep an eye out for any issues

**Pull request merged**